runtime/launcher.m

```diff
@@ -357,6 +358,8 @@
 		// So route through MONO_DEBUG
 		setenv ("MONO_DEBUG", "no-gdb-backtrace", 0);
 	}
+
+	setenv ("MONO_CFG_DIR", [monobundle_dir UTF8String], 1);
```

Comment:
Filed a bug w\ mono here - https://bugzilla.xamarin.com/show_bug.cgi?id=45279

I'll change that setenv param now.